<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator_window</class>
 <widget class="QMainWindow" name="Calculator_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="num_one">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_two">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_three">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_four">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_five">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_six">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_seven">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_eight">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_nine">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_zero">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_plus">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_slash">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_asterisk">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_equals">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_dash">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="op_clear">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_back">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_decimal">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_clickme">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>50</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Click me</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
